### PR TITLE
Refactor denied user action

### PR DESCRIPTION
**Pull request by Marina20V**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @Stanislav-Vyhnan 

### Second Level Review

- [ ] @VadymKharyton 
- [ ] @andrii-kuzmichov 

## Summary of issue
An old component, need to be refactored

## Summary of change
Refactor map modal component

## Testing approach
Testing in browser

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VadymKharyton — COMMENTED** [review]
Looks good to me.
But you need to fix build issue defined [here](https://dev.azure.com/eventsexpress/EventsExpress/_build/results?buildId=1229&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=9c939e41-62c2-5605-5e05-fc3554afc9f5&l=87). After that I can approve

File: src/components/event/DeniedUsersAction/Denied-users-action.js [hunk 1]
```diff
@@ -2,40 +2,30 @@
 import Button from "@material-ui/core/Button";
 import IconButton from "@material-ui/core/IconButton";
 import DeleteIcon from '@material-ui/icons/Delete';
-import { connect } from 'react-redux';
-import { promoteToOwner, approveUser } from '../../../actions/event/event-item-view-action';
```
Comment (andrii-kuzmichov) [inline src/components/event/DeniedUsersAction/Denied-users-action.js:5]: remove empty row

File: src/components/event/DeniedUsersAction/Denied-users-action.js [hunk 2]
```diff
@@ -2,40 +2,30 @@
...
                 </div>
             }
             <Button
-                onClick={() => props.approveUser(user.id, props.eventId, true)}
+                onClick={() => approveUser(user.id, eventId, true)}
                 variant="outlined"
                 color="success"
             >
-                Add to event
+                {constants.ADD_TO_EVENT}
```
Comment (andrii-kuzmichov) [inline src/components/event/DeniedUsersAction/Denied-users-action.js:25]: use destructuring

File: src/constants/deniedUsersAction.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const constants = {
```
Comment (andrii-kuzmichov) [inline src/constants/deniedUsersAction.js:1]: Change name

File: src/containers/DeniedUsersAction/DeniedUsersActionContainer.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { promoteToOwner, approveUser } from '../../actions/event/event-item-view-action';
+import DeniedUsersAction from '../../components/event/DeniedUsersAction/Denied-users-action';
+
```
Comment (andrii-kuzmichov) [inline src/containers/DeniedUsersAction/DeniedUsersActionContainer.js:4]: remove extra line

File: src/containers/DeniedUsersAction/DeniedUsersActionContainer.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+const DeniedUsersActionContainer = props => <DeniedUsersAction props={props} />
```
Comment (andrii-kuzmichov) [inline src/containers/DeniedUsersAction/DeniedUsersActionContainer.js:6]: it is extra code

**Pull request closed without merging**